lab_6_pipeline/pipeline.py

```diff
@@ -44,37 +92,60 @@ def __init__(self, lemma: str = "", pos: str = "", tags: str = ""):
         """
         Initializes MorphologicalTokenDTO
         """
+        self.lemma = lemma
+        self.pos = pos
+        self.tags = tags
 
 
 class ConlluToken:
     """
     Representation of the CONLL-U Token
     """
 
-    def __init__(self, text: str):
+    def __init__(self, text: str, morphological_parameters: MorphologicalTokenDTO, position: int):
         """
         Initializes ConlluToken
         """
+        self._text = text
+        self._morphological_parameters = morphological_parameters
+        self._position = position
 
     def set_morphological_parameters(self, parameters: MorphologicalTokenDTO) -> None:
         """
         Stores the morphological parameters
         """
+        self._morphological_parameters = parameters
 
     def get_morphological_parameters(self) -> MorphologicalTokenDTO:
         """
         Returns morphological parameters from ConlluToken
         """
+        return self._morphological_parameters
 
     def get_conllu_text(self, include_morphological_tags: bool) -> str:
         """
         String representation of the token for conllu files
         """
+        position = str(self._position)
+        text = self._text
+        lemma = self._morphological_parameters.lemma
+        pos = self._morphological_parameters.pos
+        xpos = '_'
+        feats = '_'
+        if include_morphological_tags:
+            feats = self._morphological_parameters.tags if self._morphological_parameters.tags else '_'
+        head = '0'
+        deprel = 'root'
+        deps = '_'
+        misc = '_'
```

Comment:
can you avoid creating some variables and return strings directly in return statement?

Reply:
maybe yes, I just can return '\t'.join([
        str(self._position),
        self._text,
        self._morphological_parameters.lemma,
        self._morphological_parameters.pos,
        '_',
        feats,
        '0',
        'root',
        '_',
        '_'
    ]) and remove all these variables from this method.
However, I think it is easier for me to set these variables in that way, which was written before.

Reply:
Why cant you remove these?

Think about variables creation process from the memory consumption point of view

Reply:
I've changed this. Done.